charts/fluid/fluid/crds/data.fluid.io_vineyardruntimes.yaml

```diff
@@ -107,14 +107,14 @@ spec:
                       Fuse. Default is not set.
                     type: object
                   image:
-                    description: Image for Vineyard Fuse Default is `vineyardcloudnative/vineyard-fluid-fuse`
+                    description: Image for Vineyard Fuse Default is `registry.cn-hangzhou.aliyuncs.com/vineyard/vineyard-fluid-fuse`
```

Comment:
I suggest you should using `registry.aliyuncs.com` to replace `registry.cn-hangzhou.aliyuncs.com`. Because registry.aliyuncs.com is the global registry, and it uses the same ip with registry.cn-hangzhou.aliyuncs.com.

Reply:
Thanks for the detail. BTW, The PR waits for https://github.com/fluid-cloudnative/fluid/pull/3749 to merge first.